source/_posts/test.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Creio que podemos colocar um post real. Talvez sobre os 7 princípios do cooperativismo.

https://www.sicredi.com.br/coop/norte/principios-do-cooperativismo/

Reply:
Great. I will add this information.